### PR TITLE
bootstrap integration (mostly

### LINKED ISSUE #60: React Bootstrap Integration
Implement React Bootstrap

### PR DESCRIPTION
**Pull request by dgoldstein1**
closes #60 

## Changes

- install bootstrap into the project
- migrate over main components to bootstrap layouts and components (note that I created https://github.com/code-for-charlottesville/housinghub/issues/66 to track this)

## How to test

- checkout the branch
- install latest dev dependencies `(cd frontend && npm install)`
- follow `frontend/README.md` file to get the app up and running"
- click around the app and make sure things look OK

Let me know if you have any questions!! happy to help 🌈 🎊

### CODE REVIEW

File: frontend/src/components/Dropdown.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React, { useState, useEffect } from "react";
-import "bulma/css/bulma.css";
+
```
Comment (billyhunt) [inline frontend/src/components/Dropdown.js:2]: Consider removing space

File: frontend/src/components/PropertySearch.js [hunk 2]
```diff
@@ -1,101 +1,115 @@
...
+    <Container>
+      <Row>
+        <Col>
+          <SearchZipCode />
+        </Col>
+        <Col>
+          <SearchHousingType />
+        </Col>
+        <Col>
+          <div className="field">
```
Comment (billyhunt) [inline frontend/src/components/PropertySearch.js:26]: Consider using bootstrap dropdown

File: frontend/src/components/PropertySearch.js [hunk 3]
```diff
@@ -1,101 +1,115 @@
...
-          className="button is-primary is-rounded"
-          onClick={() => searchProperties()}
-        >
-          Search
-        </button>
-      </div>
-    </div>
+        </Col>
+        <Col>
+          <InputField
```
Comment (billyhunt) [inline frontend/src/components/PropertySearch.js:87]: Consider using datepicker

**Comment by billyhunt** [comment]
Oh I see that you know we need to add additional components.  So as long as we plan on removing all the old css, this is great.  Sorry I should have read the PR.  :)

**Comment by dgoldstein1** [comment]
@billyhunt 

Thanks for your review! Yes, we plan on updating the remaining components with bootstrap in #66. I'm separating this out because these components are likely to change a lot with the new mocks, and we will update the bootstrap when we rewrite them in the coming weeks (instead of rewriting everything now and also rewriting everything again).

I had a few minutes this morning and pushed up changes:

- use named imports for react-bootstrap
- remove unused CSS (I think we may need some basic css for styling of the main containers)
- remove unused files and cruft-- I want to keep this repo clean if we are going to have a lot of different types of developers touching it
- merge with upstream master

**Pull request merged**